### ISSUE TITLE
dqsegdbCheckVersion is broken

### ISSUE THREAD

**Issue by duncanmmacleod** [ligovirgo/dqsegdb#83]
The `dqsegdb.apicalls.dqsegdbCheckVersion` function is broken:

```pytb
$ python -c "from dqsegdb import apicalls; print(apicalls.dqsegdbCheckVersion('https', 'segments.ligo.org', 'H1', 'DMT-ANALYSIS_READY', '1'))"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/duncan/git/dqsegdb/dqsegdb/apicalls.py", line 80, in dqsegdbCheckVersion
    version_list=result_json['version']
NameError: name 'result_json' is not defined
```

**Comment by robertbruntz** [comment]
This is addressed in the fix for https://github.com/ligovirgo/dqsegdb/issues/48 (which is actually the same issue).

**Comment by robertbruntz** [comment]
This was fixed in one of the (many) commits included in https://github.com/ligovirgo/dqsegdb/pull/87, which was merged on 2022.01.26.
